### DISCUSSION TITLE
Midori v11.4 not run windows 7 x64?

### DISCUSSION THREAD

**Discussion by rafaelholeva** [goastian/midori-desktop#88] [Q&A]
Hello, I have been using Midori 11.3.3 on Windows 7 x64 for a long time, but in the latest Midori update, the browser no longer opens, having to return to the previous version 11.3.3, has anyone experienced the same problem?

Midori v11.4 not run windows 7 x64?

**Comment by Ponchale** [comment]
Midori stopped supporting Windows 7 and 8.1, since we are using a new ESR which is 128 if you want you can use version 11.3.3.3 is the last version that supports Windows 7.
